docs/Migrating.md

```diff
@@ -13,10 +19,13 @@
      in the `Brain` GameObjects
```

Comment:
Change to 

Edit `Brain` Scriptable Objects file's `Brain Parameters` to be the same as the parameters used 
     in the `Brain` GameObjects